### DISCUSSION TITLE
Overwork docs/Makefile

### DISCUSSION THREAD

**Discussion by nlohmann** [nlohmann/json#3623] [Ideas]
This file was once a maintainer-only Makefile, but is used in the CI since #3622. We should check if we should move some targets to a `CMakeLists.txt` file to simplify integration to the CI.

**Comment by falbrechtskirchinger** [comment]
I'm adding 3 or more targets in #3553.

Edit: Nope, sorry. That would be `docs/docset/Makefile`.
